src/digitalio.py

```diff
@@ -72,6 +74,12 @@ def switch_to_input(self, pull=None):
     def deinit(self):
         del self._pin
 
+    def cleanup(self, channel=None):
```

Comment:
this is not a `digitalio` standard interface function, so it can't go here. you can hvae an atexit() or something in your own Pin files, but it can't go here